core/domain/subtopic_page_domain.py

```diff
@@ -345,7 +345,9 @@ def validate(self):
             raise utils.ValidationError(
                 'Expected language code to be a string, received %s' %
                 self.language_code)
-        if not any([self.language_code == lc['code']
-                    for lc in constants.SUPPORTED_CONTENT_LANGUAGES]):
+        if not any(
+                self.language_code == lc['code']
```

Comment:
This is not allowed by the linter, but I think we should allow this. cc @DubeySandeep

Reply:
Which two lines? Line 348-349?

Reply:
349 and 350, basically when we have a statement enclosed from two sides it doesn't make sense to have 8 space indentation. 

Actually, I have been thinking about introducing some kind of automatic formatters into our codebase since I think after introducing types into the code (which imposes few other rules to the coding style too) it might be actually quicker to introduce one formatted instead of writing arbitrary lint checks. I want to discuss it at our next core maintainers meeting.